filesmanager/processors/xapi/event_transformers/filesmanager_events.py

```diff
@@ -0,0 +1,39 @@
+"""
+Transformers for filesmanager events.
+
+Classes:
+    FilesDownloadedTransformer: Transformer for the event edunext.xblock.filesmanager.files.downloaded.
+"""
+
+from tincan import Activity, ActivityDefinition, LanguageMap, Verb
+
+from filesmanager.edxapp_wrapper.event_routing_backends import XApiTransformer, XApiTransformersRegistry
+from filesmanager.processors.xapi import constants
+
+
+@XApiTransformersRegistry.register("edunext.xblock.filesmanager.files.downloaded")
+class FilesDownloadedTransformer(XApiTransformer):
+    """
+    Transformers for event generated when an student download files from xblock.
```

Comment:
```suggestion
    Transformers for event generated when a student download files from xblock.
```
we tend to use `an` because we pronounce wrongly estudent instead of **s**tudent